modules/sc-mesh-secure-deployment/src/nats/cbma/scripts/mess/cleanup_bridge.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+
+export SCN='/sys/class/net'
+
+cleanup_bridge_if_needed()
+{
+    for I in $SCN/$MACBR_NAME/lower_*; do
+        if [ "$I" = "$SCN/$MACBR_NAME/lower_*" ]; then
+            ebtables -t nat -D OUTPUT -j "$MACBR_NAME" --logical-out "$MACBR_NAME"
+            ebtables -t nat -X "$MACBR_NAME"
+            ip link delete "$MACBR_NAME"
+        fi
+        break
+    done
+}
+
+if [ $# -ne 2 ]; then
+    >&2 echo "Usage: $0 u bat0"
+    >&2 echo "Usage: $0 l wlan0"
+    exit 1
+fi
+export L_OR_U="$1"
+export BASE_INTERFACE_NAME="$2"
+
+MAC_REGEX='[0-9a-f]{2}:[0-9a-f]{2}:[0-9a-f]{2}:[0-9a-f]{2}:[0-9a-f]{2}:[0-9a-f]{2}'
```

Comment:
Could it be:
MAC_REGEX='([[:xdigit:]]{2}:){5}[[:xdigit:]]{2}'

Reply:
I'm always forgetting about the regex hex character class, great reminder! :+1: 

The new bridge scripts are actually literal copy-paste + stripping from other "mess" scripts, so yeah, some refactoring would be nice in them. For now I will leave this task for when the MACsec key refresh is implemented properly :angel: